elements/designer-element-palette/designer-element-palette.html

```diff
@@ -96,31 +97,54 @@
 
     _onMouseDown: function(e) {
       let el = e.target;
+      // find the palette item by crawling up until we see a .element
       while (!el.classList.contains('element')) {
         el = el.parentNode;
       }
       let proxy = dragging.createDragProxy(el, true);
       this.$.proxy.innerHTML = proxy.outerHTML;
       let bounds = el.getBoundingClientRect();
+      this._updateDragProxy({
+        left: bounds.left,
+        top: bounds.top,
+        width: el.offsetWidth,
+        height: el.offsetHeight,
+      });
+      this.fire('designer-mini-nav-collapse');
       dragging.startDrag(
         el,
         bounds.left,
         bounds.top,
         e.clientX,
         e.clientY,
-        function(x, y) {
+        function(x, y, e) {
```

Comment:
Given that `e` is already an argument in the closure scope, it would be helpful for the abbreviation to be expanded. Maybe `e` is an event in both cases, but it could possibly be expanded to indicate the event type, like `dragEvent` vs `mousedownEvent`. What do you think?